src/SDKs/CognitiveServices/management/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.csproj

```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.CognitiveServices</PackageId>
 		<Description>Microsoft Azure Management Cognitive Services Library</Description>
-		<Version>3.0.0</Version>
+		<Version>3.0.1</Version>
 		<AssemblyName>Microsoft.Azure.Management.CognitiveServices</AssemblyName>
 		<PackageTags>Cognitive Services management;Cognitive Services;</PackageTags>
 		<PackageReleaseNotes>1. Regenerate the SDK by the latest version of AutoRest and Cognitive Services Management Swagger file; 2. ProvisioningState is changed to a set of constant strings from an enum. </PackageReleaseNotes>
```

Comment:
@felixwa please update PacakgeRelease notes, so that developers know what got changed in this version. You can mention updating notice.txt

Reply:
Notes added.